### DISCUSSION TITLE
A feature that lets you organize bookmarks into folders or tags

### DISCUSSION THREAD

**Discussion by Rafael-Dagostim** [dailydotdev/daily#846] [Feature Requests]
### Reason
I usually like to bookmark all different types of posts (web, backend, IOT, etc.), but when I want to find some old post I've saved, it´s a headache. You need to look at each post in the bookmark or remember a part of the title to use in the search tab.

### Solution
Allow creation of custom tags and attach then to bookmarked posts, or create folders and store these posts.

**Comment by idoshamun** [comment]
Hi @Rafael-Dagostim,

Thanks for the suggestion.
Let's keep the discussion here: https://github.com/dailydotdev/daily/discussions/520
